x11/mesa/Portfile

```diff
@@ -110,6 +111,19 @@ if {[string match *gcc* ${configure.compiler}]} {
     configure.cxxflags-append -fpermissive
 }
 
```

Comment:
We also need my patch for 32-bit builds. I submitted that upstream in https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/32658
(My branch has it too inside an LLVM-related commit.)

Reply:
Code shouldn't be indented in that patch and ifdefs are intended like `#  if` based on the surrounding code. Rebase your branch on my updated one so I can pick the changes.

Reply:
It is actually inconsistent in the source. But I can make it in a way to minimize changes (i.e. just insert needed ifdefs).